docs/user/supported_devices.rst

```diff
@@ -198,6 +198,7 @@ ipq40xx-generic
 * Linksys
 
   - EA6350 (v3)
+  - MR8300
```

Comment:
```suggestion
```

No need to try adding it as supported, as long as it's marked as broken in the target definitions.
Either you go for adding it as `broken` or for adding it as `supported`, but not both.

And as above, code reviews usually happen after a PR leaves the draft state.